### DISCUSSION TITLE
moving repo to a new folder

### DISCUSSION THREAD

**Discussion by Softbox44** [borgbackup/borg#8560] [Q&A] [answered]
I have 2 physical servers, backing up from A->B, whereby the borg repo is stored on Server B and the folder is shared via NFS. So the borg backup from A to B is a local backup as Server A has the remote nfs from Server B mounted.  
I now need to move the borg repo to another folder, so I copied the data on Server B from folder OLD to folder NEW and re-mounted the new folder as nfs. All fine.

But when I run borg it is now complaining with error:
`Cache is newer than repository - do you have multiple, independently updated repos with same ID?`

How can I force borg to accept the repo in the new folder?

**Comment by infectormp** [comment, marked as answer]
RTFM https://borgbackup.readthedocs.io/en/stable/faq.html#this-is-either-an-attack-or-unsafe-warning

**Comment by ThomasWaldmann** [comment]
Yeah, what @infectormp linked will likely solve the problem.

A bit unexpected though that just copying the repo to a new place triggers this.
